src/executable-code/index.js

```diff
@@ -101,6 +111,29 @@ export default class ExecutableCode {
       highlightOnly = true;
     }
 
+    let crosslink = null;
+
+    const isCrosslinkDisabled = (
+      highlightOnly || // highlighted only not worked in...
+      targetNode.hasAttribute(ATTRIBUTES.HIDE_CROSSLINK === 'disabled') || // disabled by developer
```

Comment:
I'm a little confused:
If the attribute with the name HIDE_CROSSLINK has a "disabled" value, it means that crosslink should be visible. But the variable name is isCrosslinkDisabled, which means the opposite.